test/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -1811,8 +1774,13 @@ public void Build_respects_PublishTrimmed_property(string targetFramework)
             // just setting PublishTrimmed doesn't inject the IsTrimmable attribute
             AssemblyInfo.Get(assemblyPath).ContainsKey("AssemblyMetadataAttribute").Should().BeFalse();
         }
+    }
+
+    internal static class ILLinkTestUtils
+    {
```

Comment:
I kept the utility functions in the same file to minimize the diff but let me know if I should move it to a separate file